### ISSUE TITLE
Add Robin Champieux to the fact-sheets or as a contributor?

### ISSUE THREAD

**Issue by daniellecrobinson** [mozillascience/open-data-training#29]
Robin contributed a bunch of feedback to the factsheets - not sure where those wound up - and should be credited as an author or acknowledged. I can check in with her about it. @stephwright will those be part of this?

**Comment by stephwright** [comment]
Yep! Added her to the list on the README and will make sure she's credited in the Instructor Guides once I get them online.  Thx for bringing it up!
